ads/google/doubleclick.js

```diff
@@ -53,6 +54,15 @@ export function doubleclick(global, data) {
     };
   }
 
+  // Center the ad in the container.
+  setStyles(global.document.querySelector('#c'), {
+    top: '50%',
+    left: '50%',
+    bottom: '',
+    right: '',
+    webkitTransform: 'translate(-50%, -50%)',
```

Comment:
Yes, but `setStyle` will correctly [map](https://github.com/ampproject/amphtml/blob/master/src/style.js#L96-L97) `transform` to `webkitTransform`.